Sources/App/Core/Gitlab.swift

```diff
@@ -139,6 +139,7 @@ extension Gitlab.Builder {
         case success
     }
 
+    // periphery:ignore
```

Comment:
Periphery seems to have trouble detecting the use of types that are only used in decoding.